api/translator/src/main/java/com/project/translator/adapter/out/persistence/MessagePersistenceAdapter.java

```diff
@@ -151,6 +151,11 @@ private void createMessageTranslation(MessageDetails messageDetails) {
         final var languageId = messageDetails.language();
         final var originalMessageId = messageDetails.original_message();
         final var originalMessage = getMessageEntity(originalMessageId);
+        final var originalMessLang = originalMessage.getLanguage();
+        if(!isLanguageEnglish(originalMessLang)) {
+            throw new OriginalMessageNotInEnglishException(originalMessLang.getLanguage());
+        }
+
```

Comment:
Przy próbie zbudowania projektu z linii komend pojawia się następujący problem:
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.607 s <<< FAILURE! - in com.project.translator.adapter.out.persistence.MessagePersistenceAdapterTest
com.project.translator.adapter.out.persistence.MessagePersistenceAdapterTest  Time elapsed: 0.607 s  <<< ERROR!
org.hibernate.LazyInitializationException: could not initialize proxy [com.project.translator.adapter.out.persistence.LanguageEntity#1] - no Session
	at com.project.translator.adapter.out.persistence.MessagePersistenceAdapterTest.setup(MessagePersistenceAdapterTest.java:63)